app/states/texas/validators/texas_settings.py

```
@@ -9,15 +12,20 @@
 ==== TEC Settings ====
 ======================
 """
-
+def check_contains_factory(match_string: str):
+    def check_contains(value: str) -> str:
+        if value and match_string not in value:
+            raise ValueError(f"Value must contain '{match_string}'")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)